lib/soapy_cake/time_converter.rb

```diff
@@ -0,0 +1,35 @@
+module SoapyCake
+  class TimeConverter
+    def initialize(time_zone, time_offset = nil)
+      if time_offset
+        self.class.print_deprecation_warning
+
+        # Etc/GMT time zones have their sign reversed
+        time_zone = format('Etc/GMT%+d', -time_offset.to_i)
+      end
+
+      fail Error, 'Cake time zone missing' if time_zone.blank?
+      @zone = ActiveSupport::TimeZone[time_zone]
+    end
+
+    def to_cake(date)
```

Comment:
What about adding a comment here that either date must have a specified timezone here or that UTC is automatically assumed? While "expect(subject.to_cake(Date.new(2015, 1, 2))).to eq('2015-01-02T01:00:00')" perfectly makes sense given the logic for other arguments, I am not sure if it might be somewhat misleading when used in the daily ETL context. What's your opinion?